Doc/library/typing.rst

```diff
@@ -678,10 +684,8 @@ The module defines the following classes, functions and decorators:
 .. class:: Dict(dict, MutableMapping[KT, VT])
 
    A generic version of :class:`dict`.
-   The usage of this type is as follows::
-
-      def get_position_in_index(word_list: Dict[str, int], word: str) -> int:
-          return word_list[word]
+   Useful for annotating return types. To annotate arguments it is preferred
+   to use an abstract collection type such as :class:`Mapping`.
```

Comment:
I would keep an example here, but update it to use a return annotation:
```python
def count_words(text: str) -> Dict[str, int]:
    ...
```

Reply:
Done.